.github/workflows/sast.yml

```
@@ -35,19 +35,19 @@ jobs:
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v2
+      uses: github/codeql-action/init@v3
 
-    - name: Setup JDK 17
-      uses: actions/setup-java@v2
+    - name: Setup JDK 21
+      uses: actions/setup-java@v4
       with:
-          java-version: 17
-          distribution: 'adopt'
+         java-version: 21
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 9

(indentation)